Header header
time start_at
time end_by
Pulse[] pulse_seq
uint8[] pins_to_signal
